src/iccpd/src/mlacp_sync_update.c

```diff
@@ -89,7 +104,17 @@ int mlacp_fsm_update_Agg_conf(struct CSM* csm, mLACPAggConfigTLV* portconf)
     {
         /*Purge*/
         if (pif != NULL )
+        {
+            //This handler would take of handling mlacp changes based on peer
+            //mclag interface delete; recover back mac of po to original
+            //on standby etc
+            mlacp_peer_mlag_intf_delete_handler(csm, pif->name);
+
+            /* Delete remote interface info from STATE_DB */
+            if (csm)
```

Comment:
This checkis not needed.   line 92 already ensure cms is not NILL

Reply:
Thanks, Done.!